split-webapp/split/src/App.js

```diff
@@ -14,7 +15,7 @@ function App() {
           <header className="App-header">
             <div>
               <Route exact path="/" component={Login} />
-              <Route path="/home" component={Home} />
+              <AuthorizedRoute path="/home" component={Home} />
```

Comment:
I'm noticing this problem:

Given the user is not logged in and has the login page `/` loaded,
When the user navigates to `/#/home/split` directly from the browser on the same tab as the loaded login page,
Then the app becomes blank.

I think this is because `/#/home` is not routed when the user is not logged in, and the `Redirect` has already been rendered and maybe there isn't anything to trigger a re-render.

This is different to loading `/#/home/split` in a new tab, where `Redirect` has yet to be rendered and redirects correctly.